### ISSUE TITLE
Allow templated access to a stable world info api from WorldListManager

### ISSUE THREAD

**Issue by Earthmark** [Neos-Metaverse/NeosPublic#2755]
### Is your feature request related to a problem? Please describe.

I want to make a world that acts as a session viewer, where you travel between worlds. However, getting access to session information is very rough right now, although some systems seem very close to being viable for this goal.

### Relevant issues

Radiant dash feedback: #727

### Describe the solution you'd like

WorldItemTemplate seems pretty valid for this purpose already, however no WorldItem exists that is useable in a generic way, the only concrete implementations are UIUX specific for the world browser.

My request is to add a concrete WorldItem that provides save (non _* named) access to the common fields of a world including:
* Thumbnail URL
* World Name
* Session Name (these can diverge to my knowledge, see a new user joining the MTC)
* Host Name
* Access Level (registered, friend, friend+...)
* Player Count
* Max Player Count
* Session ID
* World ID
* Session URL
* Session Web Url
* Open Status (if it's currently open, this is per user)

With this style of creating templated world viewers, users can then create any version of a session viewer they want, instead of being restricted to the current orb spawner or the WorldListManager's current UIUX based implementations.

### Describe alternatives you've considered

Hacking into an existing world detail panel, however that is brittle. This may be viable though, although it is still very UIUX bound.
Hacking into a session orb spawner, this is even more brittle and requires ref hacking.

**Comment by Frooxius** [comment]
We could do raw access to the items like that. There's a few things though that might make it a bit tricky perhaps though, the world items aggregate some information together - e.g. it can represent multiple sessions of the world. 

We could expose this as a list of all the sessions on the item perhaps, with aggregated info as the root level fields. Would that work well enough?

**Comment by Earthmark** [comment]
Interesting, I forgot that it provides world information...
Would it be possible to have the system unwrap all sessions? My intent would be to have each session (and not each world) be a doorway in this specific usage, I don't care about the world list, only the session list.

Perhaps have the WorldListManager be able to output either a session based view, world based view, or spliced view?
I think many usages I see of this (including one in the world I'm currently in) could utilize both a open-a-world centric view, and a join-a-session centric view.

Then have the target item contain the union of all applicable fields (including the session name that is not the world name), and just null when not applicable (when looking at a world instead of a session). Possibly have a checkbox or enum for the record kind (either session or world that could spawn a session)?

**Comment by Frooxius** [comment]
Yes, you can configure the WordListManager by setting MergeSessionsByWorldId to false.

However the WorldItem still needs to be capable of handling when they're merged, so we need to model it properly regardless.

**Comment by Earthmark** [comment]
At this moment I don't think I can access that information anyways, as it would mean indexing into a list of sessions. Perhaps the ability to view the sub sessions could be added after collections, once a consumer of this component could see elements in the sub list?
